src/Library/demos/Drawing Area/main.json

```
@@ -0,0 +1,10 @@
+{
+  "name": "Drawing Area",
+  "category": "user_interface",
+  "description": "Connection between control and drawing.",
+  "panels": [
+    "ui",
+    "preview"
```

```suggestion
    "code"
```

code is more important for this demo